src/main/java/com/basho/riak/client/core/query/timeseries/CollectionConverters.java

```diff
@@ -104,16 +110,44 @@ private static FullColumnDescription convertDescribeResultRowToFullColumnDescrip
         final ColumnDescription.ColumnType type =
                 ColumnDescription.ColumnType.valueOf(typeString.toUpperCase(Locale.ENGLISH));
 
+        final Quantum quantum = parseQuantumCells(cells);
+
         return new FullColumnDescription(name,
                                          type,
                                          isNullable,
                                          partitionKeyOrdinal,
-                                         localKeyOrdinal);
+                                         localKeyOrdinal,
+                                         quantum);
+    }
+
+    private static Quantum parseQuantumCells(List<Cell> cells)
+    {
+        final boolean hasQuantum = cells.size() > 5 && cells.get(5) != null && cells.get(6) != null;
```

Comment:
There's two different uses going on here, the line
```java
final boolean hasQuantum = cells.size() > 5 && cells.get(5) != null && cells.get(6) != null;
```
just checks to see if there's quantum information available before attempting to pull it out.

The `isValidV2Description` validates the whole row, where it's valid to have a quantum or no quantum attached to it.